go.mod

```diff
@@ -6,7 +6,6 @@ require (
 	github.com/btcsuite/btcd v0.0.0-20190115013929-ed77733ec07d
 	github.com/cosmos/go-bip39 v0.0.0-20180618194314-52158e4697b8
 	github.com/cosmos/ledger-cosmos-go v0.10.3
-	github.com/cosmos/tools/cmd/clog v0.0.0-20190722180430-ea942c183cba // indirect
```

Comment:
Why did we remove clog?

Reply:
That I don't know what's up with. I didn't do it on purpose. Maybe when running `go mod tidy`?

Is `clog` really an sdk dependency?

Reply:
it's not a dependency afaik